src/status_im2/contexts/profile/settings/header/avatar.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im2.contexts.profile.settings.header.avatar
+  (:require [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.reanimated :as reanimated]
+            [status-im2.contexts.profile.settings.header.style :as style]))
+
+(defonce ^:const scroll-animation-input-range [0 50])
```

Comment:
`defonce` is essentially a macro to help not evaluate stateful code more than once while using a REPL. Here the value is an "inlineable" const and the value always evaluates to the same value, so there's no need for `defonce`. There's another instance in this PR in file `src/status_im2/contexts/profile/settings/header/header_shape.cljs`.

Perhaps it's confusing some devs, because in the repo I've seen quite a few pointless usages of `defonce`, example: https://github.com/status-im/status-mobile/blob/6f9bcd1bb1754f7735d33e966717c82e32415c2f/src/status_im2/contexts/chat/messages/navigation/style.cljs#L6-L7

ClojureScript source: https://github.com/clojure/clojurescript/blob/8eba755046ff3ae915ed3378574486840621a491/src/main/clojure/cljs/core.cljc#L627